release-plz.toml

```diff
@@ -0,0 +1,24 @@
+[workspace]
+dependencies_update = true
+git_release_type = "auto"
+changelog_update = true
+release_always = false
```

Comment:
@nyurik this means that this has no risk of accidentally publishing a release.
-> Lets move forward with this PR as is